spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/TaskTemplate.java

```diff
@@ -283,18 +294,26 @@ public TaskExecutionResource.Page executionList() {
 	}
 
 	@Override
-	public PagedModel<TaskExecutionThinResource> thinExecutionList() {
+	public TaskExecutionThinResource.Page thinExecutionList() {
```

Comment:
Signature change? Seems breaking.

Reply:
The result is the same type.

Reply:
https://github.com/spring-cloud/spring-cloud-dataflow/blob/464868b87ab9a322a6dc7a2c9584475bbcfab8dc/spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/resource/TaskExecutionThinResource.java#L185

Reply:
Well if they are the same let's leave it as is since this is a simple backport on what will be a patch release. What is the value in changing it?

Reply:
This is a lot cleaner as a result `return restTemplate.getForObject(executionsLink.getHref(), TaskExecutionThinResource.Page.class);`